source/ViewFile.cpp

```diff
@@ -4,29 +4,54 @@
 #include "WindowPeek.h"
 #include <math.h>
 #include <stdio.h>
-
+#include <iostream>
+#include <GroupLayout.h>
 
 // external threads -- see threads.cpp for the actual functions
 extern int32 selectionWait(void *);
 extern int32 slideshowThread(void *);
 
 
 ViewFile::ViewFile(BRect R, char* name, char* path, Setup *s, Language *w)
-:  BView(R,name, B_FOLLOW_TOP_BOTTOM, B_WILL_DRAW | B_FRAME_EVENTS)
+:  BView(R,name, B_FOLLOW_TOP_BOTTOM, B_WILL_DRAW | B_FRAME_EVENTS | B_SUPPORTS_LAYOUT)
```

Comment:
When using layout management (LM), you need to use the LM aware constructor of the object, i.e. without the BRect (and here the B_FOLLOW_* flags).

Reply:
> The minimal window size seems a bit large.

This is because somehow, after each close, the app saves the window's size, then constructs the window with the same size at each opening.

The saved size becomes the **minimal** size.

Deleting the saved file resets the minimal size.